tests/chart_tests/test_astronomer_dag_only_deploy.py

```diff
@@ -193,3 +208,71 @@ def test_dagonlydeploy_config_enabled_with_persistence_delete(self, kube_version
         prod = yaml.safe_load(doc["data"]["production.yaml"])
         assert prod["deployments"]["dagOnlyDeployment"] is True
         assert persistenceRetain == prod["deployments"]["dagDeploy"]["persistence"]
+
+    def test_houston_configmap_with_dagonlydeployment_liveness_probe(self, kube_version):
```

Comment:
I think it would be find to merge these two tests into `test_houston_configmap_with_dagonlydeployment_probes` and test customizing them both in one test case.

If we're not going to do that, then we should add checks that readinessProbe does not show up when only livenessProbe is specified, and vice versa.